crap-advisor-frontend/src/app/constants/index.ts

```diff
@@ -0,0 +1,10 @@
+export const apiUrl = "http://25.38.182.18:8082";
+
+export type $needToFix = any;
```

Comment:
Is this needed now?

Reply:
Yeah, lets keep this type alias in our code. Really useful for debug typelesses variables